packages/message/src/types.ts

```diff
@@ -1,15 +1,29 @@
-export type MessageTemplate = {
-  cumulus_meta: object
+import { Message } from '@cumulus/types';
+
+export interface MessageTemplateCumulusMeta {
+  queueExecutionLimits: {
+    [queueUrl: string]: number
+  }
+}
+
+export interface MessageTemplate {
+  cumulus_meta: MessageTemplateCumulusMeta
   meta: object
-};
+}
 
-export type Workflow = {
+export interface Workflow {
   arn: string
   name: string
-};
+}
 
-export type QueueMessageMeta = {
+export interface QueueMessageMeta {
   workflow_name: string
   collection?: object
   provider?: object
-};
+}
+
+export interface CumulusQueueMessage {
+  cumulus_meta: Message.CumulusMeta & MessageTemplateCumulusMeta
+  meta: QueueMessageMeta
+  payload: object
```

Comment:
This should probably be `unknown`, since the payload doesn't have to be an object.